ci-operator/config/openshift/ironic-image/openshift-ironic-image-master.yaml

```diff
@@ -0,0 +1,34 @@
+base_images:
+  base:
+    name: "4.2"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.10
```

Comment:
Do you really want 1.10?

Reply:
just copy/paste from another example i suspect -- this image doesn't build any golang code

Reply:
no, not really, we don't really use golang in this image to be honest
any suggestion ?

Reply:
I just put 1.12